### PR TITLE
Updated mesheryctl command for applying WASM Filter

### LINKED ISSUE #11196: [Docs] Update mesheryctl command in meshery docs
### Current State
Screenshot from 2024-06-17 11-18-13
An update is need for applying the WASM filter with the appropriate command
### Desired State
Replace `mesheryctl exp filter apply --file [GitHub Link]` with 
`mesheryctl filter import [file | URL] --wasm-config [filepath|string]` 
link to file -> [here](https://github.com/meshery/meshery/blob/master/docs/pages/concepts/architectural/catalog.md)
---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Aviral0702**
Signed-off-by: Aviral0702 <aviral.asthana0704@gmail.com>

**Notes for Reviewers**

This PR fixes #11196 
Previous Behavior: 
image

Current behavior:
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d311141b7feebca2024d9815df8e2d0537cdd22d at: https://666fe40cbfcd3f0095f66aad--meshery-docs-preview.netlify.app

**Comment by Deepak0320Singhal** [comment]
@Aviral0702  Thank you for your contributions.
Let's discuss this on Websites's call on Monday. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

**Comment by vishalvivekm** [comment]
Thank you @Aviral0702.
requesting @singh1203's review.

File: docs/pages/concepts/architectural/catalog.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ It also supports a collaborative environment, where DevOps engineers can share t
 * View design file:  `mesheryctl design view [design name | ID]`
 * List all designs: `mesheryctl design list`
 5. [Importing](https://docs.meshery.io/reference/mesheryctl#cloud-native-pattern-configuration-and-management) an application. `mesheryctl design import -f [file-path]`
```
Comment (singh1203) [inline docs/pages/concepts/architectural/catalog.md:40]:
```suggestion
5. [Importing](https://docs.meshery.io/reference/mesheryctl#cloud-native-pattern-configuration-and-management) a design. `mesheryctl design import -f [file-path] -s [manifest | compose | helm]`
```

Comment (singh1203) [reply docs/pages/concepts/architectural/catalog.md:40]: @Aviral0702 Please change this accordingly.

**Review by singh1203 — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 2abbd37cc83a7530c8c8cfa39eb885251ba318de at: https://66702b886d708e0cd503c142--meshery-docs-preview.netlify.app

**Review by singh1203 — APPROVED** [review]
LGTM.

**Pull request merged**